api/scpca_portal/models/project.py

```diff
@@ -599,21 +641,30 @@ def create_computed_files(
         spatial_workflow_versions,
         multiplexed_file_mapping,
         multiplexed_workflow_versions,
-        max_workers=6,  # 6 = 2 file formats * 3 mappings.
+        max_workers=8,  # 8 = 2 file formats * 4 mappings.
         clean_up_output_data=True,
         update_s3=False,
     ):
         """Prepares ready for saving project computed files based on generated file mappings."""
 
         def create_computed_file(future):
             computed_file = future.result()
-            self.process_computed_file(computed_file, clean_up_output_data, update_s3)
+            if computed_file:
+                self.process_computed_file(computed_file, clean_up_output_data, update_s3)
 
         with ThreadPoolExecutor(max_workers=max_workers) as tasks:
             for file_format in (
                 ComputedFile.OutputFileFormats.ANN_DATA,
                 ComputedFile.OutputFileFormats.SINGLE_CELL_EXPERIMENT,
             ):
+                tasks.submit(
```

Comment:
Check if format is in the file mapping first? Then remove the check on `if computed_file` in `create_computed_file`